src/toil/leader.py

```diff
@@ -959,16 +970,39 @@ def _getStatusHint(self):
         # batch system, it will show up here as waiting to run.
         return f"{running_job_count} jobs are running, {issued_job_count - running_job_count} jobs are issued and waiting to run"
 
-    def _reportWorkflowStatus(self):
+    def _reportWorkflowStatus(self) -> None:
         """
         Report the current status of the workflow to the user.
         """
 
         # For now just log our scheduling status message to the log.
-        # TODO: make this update fast enought to put it in the progress
+        # TODO: make this update fast enough to put it in the progress
         # bar/status line.
         logger.info(self._getStatusHint())
 
+    def _write_kill_flag_file(self) -> None:
+        """
+        Write a file inside the job store that serves as a kill flag.
```

Comment:
Consider including a sentence to explain that writing `NO` is the initialized file, should only be changed when the user runs the `kill` command, and that this triggers a kill of the leader process (as opposed to a worker).